kasa/tests/test_bulb.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
-from kasa import DeviceType, SmartBulb, SmartBulbPreset, SmartDeviceException
+from kasa import BulbPreset, DeviceType, SmartDeviceException
+from kasa.iot import Bulb
```

Comment:
These would be testing the common interface in the end, but given it's named the same as the implementation class, swapping this will be just one-liner which is great :-)